frontend/src/components/comments/CommentCard.jsx

```diff
@@ -23,18 +32,31 @@ export default function CommentCard({ postId, comment, onDeleteClick }) {
     }
   }
 
+  function checkCommentUserEmail() {
```

Comment:
Could this be refactored to:

`return commentCreatorEmail === currentUser.email;`

Reply:
Yap...good one..I refactored it :)